### ISSUE TITLE
"File not found" when opening script files from file explorers

### ISSUE THREAD

**Issue by ioogithub** [massivemadness/Squircle-IDE#84]
**App Version:**
2021.1.3

**Affected Device(s):**
Android 11

**Describe the bug**
No files can be opened when selecting "Open with Squircle IDE" from several external file explorers.

**To Reproduce**
1. Navigate to a file, tap it.
2. Select "open with Squircle IDE"
3. Error is always returned: "File not found"

**Expected behavior**
File will open in in the app to be edited.

I am trying to edit bash shell scripts in the termux .shortcuts folder. I can open the files with several managers (Amaze, Material Files, Ghost Commander, Files etc.) however when I tap on the file and select open with Squircle IDE. I get an error "File not found" I have tried several files, they all result in the same error.

All of these files can be opened and edited in several other editors. Only Squircle returns this error.

**Comment by massivemadness** [comment]
Hi @ioogithub, it depends on the file explorer. 
Most of apps using the correct path `/root/storage/emulated/0/Download`,
but some apps using weird paths like this `/external_files/Download`.

I cannot guarantee that my app received the correct path from external file manager, but I can manually replace the `/external_files/` with `/storage/emulated/0/`.

**Comment by Thewisem** [comment]
Same problem here as well. And it IS from the internal storage

**Comment by massivemadness** [comment]
@Whitespace-code could you give me the names of these file explorers you tried?

**Comment by Disqu3-mirOir-qt** [comment]
any idea shen fixed , because realy anoying issue thank

**Comment by TheModernWizard** [comment]
Yup, this issue makes Squircle mostly unusable for me, using Material Files.

**Comment by lmagyar** [comment]
Also unusable on Android 8, Galaxy A3, "File not found" for everything.

**Comment by Ilithy** [comment]
I also have this problem on my smartphone (under android 12), I use the file manager of the OS (Grapheneos) only Squircle IDE reacts like this, it is indeed quite annoying as behavior, but by passing through the file browser internal to Squircle, it happens normally, for me it does not make it unusable, but it is annoying, thank you

**Comment by massivemadness** [comment]
Can anyone check the latest update [v2022.1.1](https://github.com/massivemadness/Squircle-IDE/releases/tag/2022.1.1)??
This issue is probably fixed, I tried like 10-12 different explorers and the code editor for almost all of them worked fine
